packages/flutter/test/widgets/test_border.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+typedef Logger = void Function(String caller);
+
```

Comment:
You could add "Test border that logs path accesses for testing purposes." as a description.